recipes/yanagiba/build.sh

```diff
@@ -1,4 +1,9 @@
 #!/bin/bash
 
-$PYTHON setup.py install --single-version-externally-managed --record=record.txt
```

Comment:
why have you removed this, I think this is needed: https://bioconda.github.io/linting.html#setup-py-install-args
also for the other recipe

Reply:
New recipe was generated with `conda skeleton pypi yanagiba` the build script reverted to the form without flags. 

Have added back in the latest commit.